helpers/extension/driver.js

```diff
@@ -150,20 +150,42 @@ class ExtensionProtocol extends ChromeProtocol {
     });
   }
 
-  // Stubs to bypass a page reload for now.
-  /* eslint-disable no-unused-vars */
   off(eventName, cb) {
-    return undefined;
+    if (typeof this._listeners[eventName] === 'undefined') {
+      return;
+    }
+
+    const callbackIndex = this._listeners[eventName].indexOf(cb);
+    if (callbackIndex === -1) {
+      return;
+    }
+
+    this._listeners[eventName].splice(callbackIndex, 1);
   }
 
-  gotoURL(url, waitForLoad) {
-    return Promise.resolve();
+  gotoURL(url, waitForLoaded) {
```

Comment:
should this version of `gotoURL` just be dropped in favor of using the inherited version now?